methods/channels/channels.list.json

```diff
@@ -53,5 +53,5 @@
     "internal_error": "The server could not complete your operation(s) without encountering an error, likely due to a transient issue on our end. It's possible some aspect of the operation succeeded before the error was raised.",
     "method_deprecated": "The method called is retired and no longer accepts requests. Examine the error closely for suggestions on alternative methods to use or browse api.slack.com/changelog for more information."
   },
-  "deprecated": true
+  "deprecation_warning": "This method is deprecated. It will stop functioning in February 2021 and will not work with newly created apps after June 10th, 2020. Learn more. Please use these methods instead:conversations.listusers.conversations"
```

Comment:
The raw text will have some funkiness, is that ok? In particular:
- "Learn More" is a link but is plain text here
- the list of alternative methods will not be punctuated properly (e.g. the above joins two bullet points together without a space)

I could try to format it nicer, but it felt like we may as well hardcode the text at that point >_< Maybe a good middle ground is to separate the `p` tags from the `ul`. HTML for reference:
```
div>
<p><strong>This method is deprecated. It will stop functioning in February 2021 and will not work with newly created apps after June 10th, 2020.</strong><a href="/changelog/2020-01-deprecating-antecedents-to-the-conversations-api"> Learn more</a>. </p>
<p>Please use these methods instead:</p>
<ul><li><a href="/methods/conversations.archive"><code>conversations.archive</code></a></li></ul>
</div>
```